media/src/commonMain/kotlin/dev/icerock/moko/media/FileMedia.kt

```diff
@@ -6,5 +6,6 @@ package dev.icerock.moko.media
 
 class FileMedia(
     val name: String,
-    val path: String
+    val path: String,
+    val byteArray: ByteArray
```

Comment:
store of readed file is not optimized.
you can save some additional data about reading of file to this obj, but not store `ByteArray` please - it can be large